### DISCUSSION TITLE
docs/v1.12.x/advance/vpc-internal-lb/

### DISCUSSION THREAD

**Discussion by giscus** [kubeovn/kube-ovn#3595] [Announcements]
# docs/v1.12.x/advance/vpc-internal-lb/

https://kubeovn.github.io/docs/v1.12.x/advance/vpc-internal-lb/

**Comment by WaterAndBread** [comment]
按本节文档进行实验后，确认个问题， 是不是pod 内部只可以通过 vip 进行访问，而通过 slr 创建的svc , 
  无法在pod 内部 通过 svcname.ns.svc.cluster.local 进行访问。 是否可以简要说明一下原因。谢谢。

**Reply by zbb88888** [reply]
https://kubeovn.github.io/docs/v1.12.x/advance/vpc-dns/ 这里有个测试示例

**Reply by WaterAndBread** [reply]
嗯嗯，通过 nslookup  svcname.ns.svc.cluster.local   dns  地址 ，确实是可以正常解析并返回 POD IP 地址 。但是如果直接通过 curl  http://svcname.ns.svc.cluster.local   会返回  Could not resolve host  ，这个结果是否正常。

**Reply by zbb88888** [reply]
nslookup svcname.ns.svc.cluster.local 正常的， 那 curl nslookup svcname.ns.svc.cluster.local  呢？

**Reply by WaterAndBread** [reply]
嗯，应该是我表达问题，直接在pod 里面通过 访问 svcname.ns.svc.cluster.local  ，比如 telnet   svcname.ns.svc.cluster.local   端口 ，或者 curl  http://svcname.ns.svc.cluster.local  端口，是无法通信的。因为本章节 虽然建立了svc , dns 地址还是k8s 集群内部coredns地址。 若要直接访问，需结合下一章节，自建dns ，并在deployment   加入 dnsConfig  配置，才可直接使用。
